Source/Core/Core/FreelookManager.cpp

```diff
@@ -99,47 +99,50 @@ void FreelookController::Update()
 {
   g_freelook_camera.SetControlType(Freelook::g_ActiveConfig.cameraConfig.iControlType);
 
-  if (m_move_buttons->controls[0]->GetState<bool>())
-    g_freelook_camera.MoveVertical(-g_freelook_camera.GetSpeed());
+  if (g_freelook_camera.IsActive())
```

Comment:
You can invert the conditional:
```
if (!g_freelook_camera.IsActive())
  return;
```
to avoid having to indent most of the lines in this function